### PR TITLE
Fixing scoring bug!

### PR DESCRIPTION
**Pull request by lawzlo**
@stkem 

Tested in command line. It works now.

### CODE REVIEW

**Comment by lawzlo** [comment]
scala> val a = Map("foo"->3, "raise"->1, "the"->2, "bar"->4).toList.sortWith(_._2 > _._2)
a: List[(String, Int)] = List((bar,4), (foo,3), (the,2), (raise,1))

scala> a.take(500).map { case(uriId, count) => (uriId, count.toDouble/a.head._2)}
res1: List[(String, Double)] = List((bar,1.0), (foo,0.75), (the,0.5), (raise,0.25))

File: kifi-backend/modules/graph/app/com/keepit/graph/commanders/GraphCommander.scala [hunk 1]
```diff
@@ -17,18 +17,18 @@ class GraphCommander @Inject() (
   private val maxResults = 500
 
   private def getUriScoreList(vertexKind: String, vertexId: Long, journal: TeleportationJournal, avoidFirstDegreeConnections: Boolean): Seq[ConnectedUriScore] = {
-    val ls = collisionCommander.getUris(vertexKind, vertexId, journal, avoidFirstDegreeConnections).toList.sortBy(_._2)
+    val ls = collisionCommander.getUris(vertexKind, vertexId, journal, avoidFirstDegreeConnections).toList.sortWith(_._2 > _._2)
```
Comment (yingjieMiao) [inline kifi-backend/modules/graph/app/com/keepit/graph/commanders/GraphCommander.scala:20]: this sortWith looks odd. another way is `sortBy(-1f * _._2)`

Comment (lawzlo) [inline kifi-backend/modules/graph/app/com/keepit/graph/commanders/GraphCommander.scala:20]: Is there a better way to sort in decrease?

Comment (stkem) [inline kifi-backend/modules/graph/app/com/keepit/graph/commanders/GraphCommander.scala:20]: Like Yingjie said, you can just sort by the negative.

**Pull request merged**